docs/source/json-api/index.rst

```diff
@@ -165,34 +136,58 @@ token.  The default "header" is fine.  Under "Payload", fill in:
       }
     }
 
-Keep in mind that the value of the ``ledgerId`` payload field has to match the one passed to the sandbox with the ``--ledgerid`` argument. You can replace ``Alice`` with whatever party you want to use.
+The value of the ``ledgerId`` field has to match the ``ledgerId` of your underlying DAML Ledger.
+For the Sandbox this corresponds to the ``--ledgerid MyLedger`` flag.
+
+.. note:: The value of ``applicationId`` will be used for commands submitted using that token.
+
+The value for ``actAs`` is specified as a list and you provide it with the party that you want to use.
+Such as the example which uses ``Alice`` for a party. Each request can only be for one party.
+For example you couldn't have ``actAs`` defined as ``["Alice", "Bob"]``.
 
-Under "Verify Signature", put ``secret`` as the secret (*not* base64
-encoded); that is the hardcoded secret for testing.
+For the Sandbox any string will create use a party with that name if it exists, 
+or create it if it does not. This is known as "implicit party allocation" and is unique to the Sandbox.
+
+For non-Sandbox ledgers the party must already exist on the ledger in order to be used.
+This is known as "explicit party allocation" and is common on all non-Sandbox ledgers.
```

Comment:
```suggestion
The party should reference an already allocated party.
```
This is really not the right place to explain implicit party allocation. If you want to document this, add something to the sandbox docs and reference it here but I recommend leaving that to a separate PR.